src/main/java/creativei/entity/Inquiry.java

```diff
@@ -63,6 +59,16 @@ public Inquiry(InquiryVo inquiryVo) throws Exception {
     private AreaOfInterest areaOfInterest;
     @OneToOne(cascade=CascadeType.ALL)
     private InquiryAddress inquiryAddress;
+    @OneToOne(cascade = CascadeType.ALL)
```

Comment:
Should be manytoOne. An inquiry can have multiple education qualification records